statsmodels/datasets/tests/test_utils.py

```diff
@@ -51,6 +57,9 @@ def test_webuse_pandas():
     internet_available = check_internet(base_gh)
     if not internet_available:
         pytest.skip('Unable to retrieve file - skipping test')
-    res1 = webuse('macrodata', baseurl=base_gh)
+    try:
```

Comment:
does this render the internet_unavailable check above unnecessary?

Reply:
Not exactly. They will return different skip messages.

Reply:
No, since you get a different message depending on the reason.